docs/file-reference.md

```diff
@@ -883,6 +895,73 @@ secrets:
 
 Anything [EnvVarSource Spec](https://kubernetes.io/docs/api-reference/v1.8/#envvarsource-v1-core) from Kubernetes can be included within the Kedge file.
 
+## buildConfigObject
+
+```yaml
+buildConfigs:
+- <buildConfigObject>
+```
+
+> Example
+
+```yaml
+name: ruby
+buildConfigs:
+- triggers:
+  - type: "ImageChange"
+  source:
+    type: "Git"
+    git:
+      uri: "https://github.com/openshift/ruby-hello-world"
+...
```

Comment:
it would be better to show functional buildConfig here or at least one that can be created. This is just portion of it that can't be even created because it doesn't have all required fields

Reply:
done